src/segmentmytiff/logging_config.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import sys
+import time
+from contextlib import contextmanager
+from pathlib import Path
+from logging import handlers
+
+import numpy as np
+import pandas as pd
+
+
+def setup_logger(name: str = None, level=logging.INFO):
+    """
+    Setup a logger with a specific name and logging level.
+    """
+
+    path = Path('log')
+    path.mkdir(exist_ok=True, parents=True)
```

Comment:
Consider expose the path of log file as an option?

Reply:
I thought about that, but I think that adds unnecessary complexity for now. If you think we need it (now or soon), especially from the plugin side, I'll go for it of course.